qt/python/mantidqt/widgets/algorithmselector/test/test_algorithmselector.py

```diff
@@ -148,19 +151,29 @@ def test_search_box_selection_wrong_text(self):
         self.assertEqual(widget.search_box.currentText(), 'abc')
 
     def test_execute_on_click(self):
-        with patch('mantidqt.interfacemanager.InterfaceManager.createDialogFromName') as createDialog:
+        with patch(createDialog_func) as createDialog:
             widget = AlgorithmSelectorWidget()
             self._select_in_tree(widget, 'DoStuff v.2')
             widget.execute_button.click()
             createDialog.assert_called_once_with('DoStuff', 2)
 
     def test_execute_on_return_press(self):
-        with patch('mantidqt.interfacemanager.InterfaceManager.createDialogFromName') as createDialog:
+        with patch(createDialog_func) as createDialog:
             widget = AlgorithmSelectorWidget()
             self._select_in_tree(widget, 'DoStuff v.2')
             QTest.keyClick(widget.search_box, Qt.Key_Return)
             createDialog.assert_called_once_with('DoStuff', 2)
 
+    def test_execute_on_double_click(self):
```

Comment:
Nice test :+1:

Reply:
Could the word "execute" be removed from these test names?

This would avoid any confusion with actually executing/running an algorithm.